go-controller/cmd/ovnkube-trace/ovnkube-trace.go

```diff
@@ -480,6 +480,44 @@ func getPodInfo(coreclient *corev1client.CoreV1Client, restconfig *rest.Config,
 	return podInfo, err
 }
 
+func getOvnSSL(coreclient *corev1client.CoreV1Client, restconfig *rest.Config, ovnNamespace string) (bool, error) {
+	appLabel := "app=ovnkube-master"
+	nameLabel := "name=ovnkube-master"
+
+	listOptions := metav1.ListOptions{
+		LabelSelector: appLabel,
+	}
+	pods, err := coreclient.Pods(ovnNamespace).List(context.TODO(), listOptions)
+	if err != nil || len(pods.Items) == 0 {
+		klog.V(5).Infof("Cannot find ovnkube pods by label %s", appLabel)
+		klog.V(5).Infof("Retrying with label %s", nameLabel)
+
+		listOptions := metav1.ListOptions{
+			LabelSelector: nameLabel,
+		}
+		pods, err = coreclient.Pods(ovnNamespace).List(context.TODO(), listOptions)
+		if err != nil || len(pods.Items) == 0 {
+			klog.V(5).Infof("Cannot find ovnkube pods by label %s", nameLabel)
+			return false, err
+		}
+	}
+
+	masterPodName := pods.Items[0].Name
+	klog.V(5).Infof("Found master pod %s", masterPodName)
+
+	psCmd := "ps aux | egrep -o 'nb-address( |=)(tcp|ssl)' | awk -F ' |=' '{print $NF}'"
+	hostOutput, hostError, err := execInPod(coreclient, restconfig, ovnNamespace, masterPodName, "ovnkube-master", psCmd, "")
```

Comment:
@andreaskaris maybe we should do this *always* and pull out both (a) the protocol and (b) the ports? Like make this function `getDBURIs()` or something like that.

Reply:
Done